docker_build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if ! ( which docker ); then
+    echo "Docker not installed, unable to continue"
+    exit 1
+fi
+
+REPO_ROOT=$(pwd)
+PACKAGE_NAME="um-kernel"
+DOCKER_IMAGE_NAME="${PACKAGE_NAME}_build"
+DOCKER_CONTAINER_NAME="kernel_build"
+
+# Initialize repositories
+git submodule init
+git submodule update
+
+# Start clean
+sudo rm -rf ./_build_armhf 2> /dev/null
+sudo rm ./${PACKAGE_NAME}-*.deb 2> /dev/null
```

Comment:
this highly depends on how we move out artifacts. I would suggest to just wipe *.deb, but besides that, _build_armhf and thus also the deb should never be part of the docker image. If this gets pulled into the context because it (pre) existed on the outside, .dockerignore solves that. if it is committed to the image, something else goes wrong. as the whole point of docker is to be completely isolated and clean.